OracleSOASuite/dockerfiles/12.2.1.3/Dockerfile

```diff
@@ -31,20 +30,34 @@ ENV FMW_JAR1=fmw_12.2.1.3.0_soa.jar \
 # -----------------------------------
 ADD  $FMW_JAR1 $FMW_JAR2 /u01/
 COPY container-scripts/* /u01/oracle/dockertools/
+COPY install/* /u01/
+
+USER root
 RUN  yum install -y hostname &&    \
```

Comment:
The following 2 lines can be removed from here as these are duplicate and we need to run the yum install later.

Reply:
Referring to 

RUN  yum install -y hostname &&    \
     rm -rf /var/cache/yum &&      \